src/scripts/temporaryBattle/TemporaryBattleList.ts

```diff
@@ -1,6 +1,93 @@
 const TemporaryBattleList: { [battleName: string]: TemporaryBattle } = {};
 
 //Kanto Temporary Battles
+TemporaryBattleList['Blue 1'] = new TemporaryBattle(
+    'Blue 1',
+    [
+        new GymPokemon('Pidgey', 1040, 9),
+        new GymPokemon('Charmander', 1678, 9, new StarterRequirement(GameConstants.Region.kanto, GameConstants.Starter.Grass)),
+        new GymPokemon('Squirtle', 1678, 9, new StarterRequirement(GameConstants.Region.kanto, GameConstants.Starter.Fire)),
+        new GymPokemon('Bulbasaur', 1678, 9, new StarterRequirement(GameConstants.Region.kanto, GameConstants.Starter.Water)),
+        new GymPokemon('Bulbasaur', 1678, 9, new StarterRequirement(GameConstants.Region.kanto, GameConstants.Starter.Special)),
```

Comment:
Shouldn't this be Eevee?

Reply:
No. We use the same team as if Squirtle had been picked, if Pikachu was picked. Maybe that should be changed in future, but it's how it is for the Champion battle, at least.